examples/example.js

```diff
@@ -1,11 +1,51 @@
-import { Editor } from './../index'
+import {EventManager} from './../index'
 import './styles.css'
 
-const editor = new Editor()
-editor.addEvent('result', result => {
-  document.getElementById('note-textarea').value = result;
-})
+const eventManager = new EventManager();
 
-document.getElementById('start-record-btn').addEventListener("click", () => editor.start())
+//with nested events
+eventManager.addEvent(
+    "Say Hello",
+    null,
+    {
+        "nestedEvents": [
+            {
+                "command": "world",
+                "actions": [function () {
+                    document.getElementById('note-textarea').value = "Hello, world!";
+                }]
+            },
+            {
+                "command": "guys",
+                "actions": [function () {
```

Comment:
Why our action for parent event is single but here there`re array of actions ?

Reply:
Because, the there should be correspondence between data model that client posts and EventManager receives.
In this case, EventManager expects 'actions' parameter.
As for me, it is bad idea to send params in such way.
I'll think on better solutions.